cross/sudo/Makefile

```diff
@@ -0,0 +1,25 @@
+PKG_NAME = sudo
+PKG_VERS = 1.8.12
+PKG_VERS_CUT = $(PKG_VERS)
```

Comment:
Just wanted to stick to the model but yes PKG_DIR = $(PKG_NAME)-$(PKG_VERS) is shorter.